create_app.py

```diff
@@ -536,7 +616,7 @@ async def get_user_data(token_payload: UserData, user_id: int | str):
     @requires_auth(config, ["post:user"])
     async def add_user(token_payload):
         # Gets the user's data
-        user_data = json.loads(await request.data)
+        user_data = await request.get_json()
```

Comment:
does this work? It looks suspiciously like this pitfall? https://quart.palletsprojects.com/en/latest/tutorials/asyncio.html#common-pitfalls

Reply:
yup. It's their awaitable method 😄 the only issue is it crashes when there's no valid JSON, so I'm not sure it's actually a good idea...